Xamarin.Forms.Platform.WinRT/VisualElementRenderer.cs

```diff
@@ -176,6 +182,25 @@ protected override Windows.Foundation.Size ArrangeOverride(Windows.Foundation.Si
 				Rectangle bounds = child.Bounds;
 
 				renderer.ContainerElement.Arrange(new Rect(bounds.X, bounds.Y, Math.Max(0, bounds.Width), Math.Max(0, bounds.Height)));
+
+				arrangedChildren.Add(renderer.ContainerElement);
```

Comment:
Um....weird, I remember doing this after getting the feedback. Let me check my local.